src/content/en/2020/compression.md

```diff
@@ -28,61 +28,86 @@ unedited: true
-
-
-Here are some general recommendations on what compression levels to use:
+Since precompression is done beforehand, we can spend more time compressing the assets. For dynamically compressed resources, we need to choose the compression levels such that compression takes less time than the time difference between sending an uncompressed versus a compressed file. This difference is borne out when looking at compression level recommendations for both methods.
 
 <figure markdown>
-|                         | brotli  | gzip            |
-| ----------------------- | ------- | --------------- |
-| precompressed           |  11     |   9 or zopfli   |
-| dynamically compressed  |   5     |   6             |
-
+  <table>
+    <thead>
+      <tr>
+        <th></th>
+        <th>brotli</th>
+        <th>gzip</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>precompressed</td>
+        <td class="numeric">11</td>
+        <td class="numeric">9 or zopfli</td>
+      </tr>
+      <tr>
+        <td>dynamically compressed</td>
+        <td class="numeric">5</td>
+        <td class="numeric">6</td>
+      </tr>
+    </tbody>
+  </table>
 <figcaption>{{ figure_link(caption="Recommended compression levels to use.") }}<figcaption>
 </figure>
 
-Currently, when compression is used, the split between brotli and gzip is about 23% / 77%.
+Currently, practically all text compression is done by one of two HTTP content encodings: [gzip](https://tools.ietf.org/html/rfc1952) and [brotli](https://github.com/google/brotli). Both are widely supported by browsers: [can I use brotli](https://caniuse.com/?search=brotli)/[can I use gzip](https://caniuse.com/?search=gzip)
 
-Approximately 60% of HTTP responses are delivered with text-based compression. This may seem like a surprising statistic, but keep in mind that it is based on all HTTP requests in the dataset. Some content, such as images, will not benefit from these compression algorithms. The table below summarizes the percentage of requests served with each content encoding.
+When you want to use gzip, consider using [Zopfli](https://en.wikipedia.org/wiki/Zopfli), which generates smaller gzip compatible files. This should be done especially for precompressed resources, since here the greatest [gains are expected](https://cran.r-project.org/web/packages/brotli/vignettes/brotli-2015-09-22.pdf). See this [comparison between gzip and zopfli](https://blog.codinghorror.com/zopfli-optimization-literally-free-bandwidth/) that takes into account different compression levels for gzip.
+
+Many [popular servers](https://en.wikipedia.org/wiki/HTTP_compression#Servers_that_support_HTTP_compression) support dynamically and/or pre-compressed HTTP and many of them support [brotli](https://en.wikipedia.org/wiki/Brotli).
+
+## Current state of HTTP compression
+
+{# TODO(authors): The below graph would imply that this should be "no" compression, reversing the meaning. Is that accurate? #}
```

Comment:
@mo271

Reply:
I don't quite get the question, but the text seems to be accurate.

Reply:
Sorry I should be more explicit. 

The old text said "Approximately 60% of HTTP responses are delivered with text-based compression." But the data table below had the exact opposite data in it. So I changed that text by adding the "no". 

We should be good now. I'll delete the TODO.